source/beamline/REIXS/REIXSXESMCPDetector.cpp

```diff
@@ -343,3 +371,13 @@ void REIXSXESMCPDetector::acquisitionCancelledHelper(){
 
 	emit imageDataChanged();
 }
+
+void REIXSXESMCPDetector::toggleVeto(bool on)
+{
+	if (vetoControl_->isConnected() && vetoStateControl_->isConnected()) {
+		if (on)
+			vetoControl_->move(0); // veto off is to enable updating
+		else
+			vetoControl_->move(1); // veto on is to disable updating
+	}
+}
```

Comment:
I feel like the semantics of this are backwards.
veto suggests stopping updates, which we can see in the if statement of this function is currently semantically reversed to the pv values. 
```
void REIXSXESMCPDetector::toggleVeto(bool on) {
    if(on)
        move veto pv to off
    else
        move veto pv to on
}
```

the result of this is that all the calls in the rest of the class also seem backwards eg.

```C++
void REIXSXESMCPDetector::resumeDwelling()
{
    if (dwellTimeTimer_) {
        dwellTimeTimer_->resume();
        toggleVeto(true);
     }
}
```
If we're resuming dwelling, it makes sense to turn the veto off.
The logic of the if statement needs to be reversed in ```toggleVeto```, and the boolean values need to be reversed in all the calls to ```toggleVeto```